### DISCUSSION TITLE
How can i achieve spacing in edge and node

### DISCUSSION THREAD

**Discussion by augusto3691** [xyflow/xyflow#2677] [Q&A]
I saw in some examples the 'smoothstep' has this behavior:

![image](https://user-images.githubusercontent.com/8695590/207358498-2b5bfc45-a31d-4d9c-8ddf-53920ef49652.png)

See that space between node and first curve in path? How can i achieve this?
In my example i got this:

![image](https://user-images.githubusercontent.com/8695590/207358742-90bc948a-4e27-4dc6-9c9c-fc9becdc4a92.png)

They are connected correctly but is like underneth nodes, in [this example](https://reactflow.dev/docs/examples/misc/download-image/) that happens but i can't find what it's is diferent from [that](https://reactflow.dev/docs/examples/nodes/custom-node/) for example.

I change nothing in the example code but cant achieve this little spacing

**Comment by moklick** [comment]
which version are you using?

**Reply by augusto3691** [reply]
^10.2.3

**Reply by augusto3691** [reply]
something with version?

**Reply by moklick** [reply]
Yep. Please upgrade to v11!

**Reply by augusto3691** [reply]
missing something in V11? I made all changes from guide

```sheel
Failed to compile.

./node_modules/@reactflow/minimap/dist/esm/index.js 100:29
Module parse failed: Unexpected token (100:29)
File was processed with these loaders:
 * ./node_modules/react-scripts/node_modules/babel-loader/lib/index.js
You may need an additional loader to handle the result of these loaders.
|     nodeOrigin
|   } = useStore(selector, shallow);
>   const elementWidth = style?.width ?? defaultWidth;
|   const elementHeight = style?.height ?? defaultHeight;
|   const nodeColorFunc = getAttrFunction(nodeColor);
```

My react version is `^17.0.2`
building with yarn and npm, same thing
